apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -72,7 +74,7 @@ class ReviewApplication extends React.Component {
             >
               {application.user.name ? application.user.name : application.user.login}
             </SafeLink>
-            applied 8 days ago
+            applied {etaAgo} ago
```

Comment:
The eta should be the estimated time of completion that's different from when it's submitted or when they apply right?